content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -255,6 +252,11 @@ different Kubernetes components.
 | `CSIMigrationAzureDiskComplete` | - | Deprecated | 1.21 | - |
 | `CSIMigrationAzureFileComplete` | `false` | Alpha | 1.17 | 1.20 |
 | `CSIMigrationAzureFileComplete` | - | Deprecated |  1.21 | - |
+| `CSIMigrationGCE` | `false` | Alpha | 1.14 | 1.16 |
+| `CSIMigrationGCE` | `false` | Beta | 1.17 | 1.22 |
+| `CSIMigrationGCE` | `true` | Beta | 1.23 | |
+| `CSIMigrationGCE` | `true` | Beta | 1.23 | |
```

Comment:
Line 256 and 257 are the same?  Also should we have 1.24 as the end release for beta?

Reply:
done. Hooray copy-paste!